test/unit/sio.c

```diff
@@ -37,6 +37,10 @@
 #define UNIT_TAP_COMPATIBLE 1
 #include "unit.h"
 
+#ifndef is_str
+#define is_str(a, b, fmt, args...) ok(strcmp(a, b) == 0, fmt, ##args)
```

Comment:
Since it's already the second place, where we need `is_str`, let's move it to the `unit.h`

Reply:
Done